### ISSUE TITLE
sometimes dynamips process doesn't stop when project closes

### ISSUE THREAD

**Issue by ehlers** [GNS3/gns3-gui#2178]
GNS3 version 2.0.4dev1 on Darwin (64-bit) with Python 3.5.2 Qt 5.6.0 and PyQt 5.6, official GNS3 VM with latest 2.0.4dev1.

Topology, all nodes (including switch) are running in the GNS3VM:
![test](https://user-images.githubusercontent.com/896406/28458937-c8611b62-6e0b-11e7-8df9-10b63c899477.png)

Only alpine-2 is turned on.

When opening another project, the dynamips processes should end. But in some cases like the above project, one dynamips process remains active.

Before closing the project:
```
gns3@gns3vm:~$ ps ax | grep dynamips
12493 ?        Sl     0:00 /usr/bin/dynamips -N1 -l dynamips_i52_log.txt -H 57469
12512 ?        Sl     0:00 /usr/bin/dynamips -N1 -l dynamips_i53_log.txt -H 35600
```

After closing the project:
```
gns3@gns3vm:~$ ps ax | grep dynamips
12512 ?        Sl     0:00 /usr/bin/dynamips -N1 -l dynamips_i53_log.txt -H 35600
```

It this situation the local server/controller won't stop gracefully, it has to be killed. The dynamips process will keep running.

**Comment by noplay** [comment]
Don't know why but the ethernet switch can start his own hypervisor:
https://github.com/GNS3/gns3-server/blob/master/gns3server/compute/dynamips/nodes/ethernet_switch.py#L112

**Comment by ehlers** [comment]
I assume, this issue might be related to https://github.com/GNS3/gns3-gui/issues/2177

My current theory is, that sending to a port, that's down, locks up the switch. Then he doesn't forward further packets (https://github.com/GNS3/gns3-gui/issues/2177) and doesn't respond to close requests.

**Comment by grossmj** [comment]
This issue is fixed by Dynamips version 0.2.17 (to be released soon).
